chapters/en/chapter12/2.mdx

```diff
@@ -0,0 +1,122 @@
+# Fundamentals of Inference for LLMs 
+
+Large Language Models (LLMs) have revolutionized how we interact with and produce text. They transform static knowledge into dynamic text generation through a process known as inference. 
+
+In this chapter, we will explore the fundamental concepts and techniques behind LLM inference, providing a comprehensive understanding of how these models generate text.
+
+## What is LLM Inference?
+
+Let's start with the basics. Inference is the process of using a trained LLM to generate human-like text from a given input prompt. At its core, the model leverages learned probabilities from billions of parameters to predict and generate the next token in a sequence, one token at a time. The model is able to predict the next token based on the previous tokens in the sequence.
+
+## Attention is All You Need
```

Comment:
a bit out of place since we talk about it earlier :p - i.e. looks a bit incoherent